generator/generator.yml

```
@@ -752,3 +752,17 @@ modules:
   hrSWInstalled:
     walk:
       - hrSWInstalled
+
+#
+# Powercom MIBs
+#
+# https://www.upspowercom.com/Card-DA807.jsp
+#
+  powercom:
+    walk:
+      - 1.3.6.1.4.1.935.1.1.1.1
+      - 1.3.6.1.4.1.935.1.1.1.2
+      - 1.3.6.1.4.1.935.1.1.1.3
+      - 1.3.6.1.4.1.935.1.1.1.4
+      - 1.3.6.1.4.1.935.1.1.1.9
+      - 1.3.6.1.4.1.476.1.42.3
```

This is unrelated to the specified MIB. It's part of `LIEBERT-GP-ENVIRONMENTAL-MIB`.

```suggestion
      - 1.3.6.1.4.1.476.1.42.3
```
```suggestion
```

---

Based on OID table at  https://www.upspowercom.com/Card-DA807.jsp this OID is exposed by the device, so does it mean if it's present in any other MIB I don't need to place it for my set of MIBs ?

---

Yes, devices can output data for multiple MIBs. We prefer to make configuration blocks here that target subsections of data and then you can scrape devices with multiple targeted modules.